### DISCUSSION TITLE
About the use of adversarial patch

### DISCUSSION THREAD

**Discussion by Zengyf-CVer** [Trusted-AI/adversarial-robustness-toolbox#1285] [Q&A] [answered]
Regarding this part of the adversarial patch, there are only examples of `TF1.x and TF2.x` in the instructions. Is there any example code for `the PyTorch version`?
![image](https://user-images.githubusercontent.com/41098760/130723260-61fa7a0d-8b2c-4e22-8582-0236563d9e2b.png)
![image](https://user-images.githubusercontent.com/41098760/130723265-2d5b3bcb-6b1e-4e65-852c-7216448709cb.png)

There is another question, is there any related papers on adversarial patch?

**Comment by beat-buesser** [comment, marked as answer]
Hi @Zengyf-CVer There is no separate notebook for the PyTorch version yet, but creating adversarial patches with PyTorch is very similar or almost identical as in the notebooks for TensorFlow. The main changes are to use `PyTorchClassifier` and `AdversarialPatchPyTorch` instead.

**Reply by Zengyf-CVer** [reply]
Thank you very much for your reply.

**Reply by beat-buesser** [reply]
Great, let us know how it goes and if you have further questions!
